sonar-project.properties

```diff
@@ -52,7 +52,10 @@ sonar.coverage.exclusions=\
   src/components/VideoPlayer/VideoPlayer.tsx, \
   src/components/VideoPlayer/index.ts, \
   src/components/CMSImage/index.ts, \
-  src/components/PupilViews/**,\
+  src/components/PupilViews/PupilIntro/**, \
```

Comment:
I don't think Quiz should be on this list anymore. It was supposed to have coverage